libs/components/tabs/src/lib/modules/tabs/tab-button.component.ts

```diff
@@ -34,7 +36,13 @@ const DEFAULT_ELEMENT_ROLE = 'tab';
 })
 export class SkyTabButtonComponent implements AfterViewInit, OnDestroy {
   @Input()
-  public active: boolean;
+  public get active(): boolean {
+    return this.#_isActive;
+  }
+  public set active(value: boolean) {
```

Comment:
`boolean | undefined`

Reply:
@Blackbaud-SteveBrush I updated this for disabled but this is an internal component we are rendering from the tabset component and the value we pass for active will _always_ be true or false--making it undefined just creates a path that could never happen and therefore is untestable.